Rubberduck.Parsing/Annotations/IdentifierAnnotationService.cs

```diff
@@ -16,24 +16,7 @@ public IdentifierAnnotationService(DeclarationFinder declarationFinder)
 
         public IEnumerable<IAnnotation> FindAnnotations(QualifiedModuleName module, int line)
         {
-            var annotations = new List<IAnnotation>();
-            var moduleAnnotations = _declarationFinder.FindAnnotations(module).ToList();
-            // VBE 1-based indexing
-            for (var currentLine = line - 1; currentLine >= 1; currentLine--)
-            {
-                //Identifier annotation sections end at the first line above without an identifier annotation.
-                if (!moduleAnnotations.Any(annotation => annotation.QualifiedSelection.Selection.StartLine <= currentLine
-                                                            && annotation.QualifiedSelection.Selection.EndLine >= currentLine
-                                                            && annotation.AnnotationType.HasFlag(AnnotationType.IdentifierAnnotation)))
-                {
-                    break;
-                }
-
-                var annotationsStartingOnCurrentLine = moduleAnnotations.Where(a => a.QualifiedSelection.Selection.StartLine == currentLine && a.AnnotationType.HasFlag(AnnotationType.IdentifierAnnotation));
-
-                annotations.AddRange(annotationsStartingOnCurrentLine);
-            }
-            return annotations;
+            return _declarationFinder.FindAnnotations(module, line).Where(annotation => annotation.AnnotationType.HasFlag(AnnotationType.IdentifierAnnotation));
```

Comment:
Nice! Should this method be inlined at this point?

Reply:
That would make sense. Then, the entire class is needed no longer.